_docs/reference/istioctl-autogen/istioctl.md

```diff
@@ -0,0 +1,37 @@
+---
+category: Reference
+title: Istioctl
+bodyclass: docs
```

Comment:
Added `overview` with the same contents as `title`. I used _docs/reference/contribute/ and didn't see `index` in any of that markdown. 

Created https://github.com/istio/istio.github.io/issues/70 to track moving the content.